Chart: update Axis Label Overlapping demo description (#2272)
DevExpress/devextreme-demos: JSDemos/Demos/Charts/AxisLabelsOverlapping/description.md

> In this demo, argument axis labels overlap due to their length. To resolve overlapping, you can specify one of the [label](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/label/).[overlappingBehavior](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/label/#overlappingBehavior) modes:

Reviewer: ```suggestion
In this demo, argument axis labels overlap due to their length. To avoid overlapping labels, you can use one of the following [label](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/label/).[overlappingBehavior](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/label/#overlappingBehavior) modes:
```

> In this demo, argument axis labels overlap due to their length. To control overlapping behavior, you can use one of the following [label](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/label/).[overlappingBehavior](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/label/#overlappingBehavior) modes:

Reviewer: ```suggestion
In this demo, argument axis labels overlap due to their length. To specify the overlapping behavior, you can use one of the following [label](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/label/).[overlappingBehavior](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/label/#overlappingBehavior) modes:
```

> - `none`    

Reviewer: This item does not suit either 'resolving' or 'avoiding' overlapping labels since it leaves overlapping lables

Leaves axis labels overlapped.

- `hide`    
Hides certain axis labels and leaves more space for the others.

- `rotate`     
Rotates axis labels at the angle specified by the [rotationAngle](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/label/#rotationAngle) property.

- `stagger`      

> Arranges axis labels in two rows in a staggered manner. Use the [staggeringSpacing](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/label/#staggeringSpacing) property to specify an empty space between rows.

Reviewer: ```suggestion
Arranges axis labels into two staggered rows. Use the [staggeringSpacing](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/label/#staggeringSpacing) property to specify an empty space between rows.
```
